test/pbelutest.c

```diff
@@ -46,7 +46,7 @@ static int test_pbelu(void)
         last_type = pbe_type;
         last_nid = pbe_nid;
     }
-    return failed ? 0 : 1;
+    return 0;
```

Comment:
The assignment on line 43 seems redundant. Side note: I am slightly confused by this logic:

    ok ? "ERROR" : "OK"